media-fonts/fonts-meta/fonts-meta-2.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Meta package for fonts to cover most needs"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cjk emoji latin ms"
+
+LANGS="as bn bo brx doi dz gu hi km kn kok ks mai ml mr ne or pa ru sa sat sd si syc ta te th"
+for lang in ${LANGS}; do
+	IUSE+=" l10n_${lang}"
+done
+unset lang LANGS
+
+RDEPEND="media-libs/fontconfig
+	cjk? (
+		|| (
+			media-fonts/mikachan-font-otf
+			media-fonts/mikachan-font-ttf
+		)
+		media-fonts/ipaex
+		media-fonts/ja-ipafonts
+		media-fonts/koruri
+		media-fonts/monafont
+		media-fonts/nanum
+		media-fonts/noto-cjk
+		media-fonts/opendesktop-fonts
+		media-fonts/shinonome
+		media-fonts/unfonts
+		media-fonts/wqy-microhei
+		media-fonts/wqy-zenhei
+	)
+	emoji? (
+		|| (
+			media-fonts/noto-emoji
+			media-fonts/joypixels
+		)
+	)
+	l10n_as? ( media-fonts/lohit-assamese )
+	l10n_bn? ( media-fonts/lohit-bengali )
+	l10n_bo? ( media-fonts/tibetan-machine-font )
+	l10n_brx? ( media-fonts/lohit-devanagari )
+	l10n_doi? ( media-fonts/lohit-devanagari )
+	l10n_dz? ( media-fonts/dzongkha-fonts )
+	l10n_gu? ( media-fonts/lohit-gujarati )
+	l10n_hi? ( media-fonts/lohit-devanagari )
+	l10n_km? ( media-fonts/khmer )
+	l10n_kn? ( media-fonts/lohit-kannada )
+	l10n_kok? ( media-fonts/lohit-devanagari )
+	l10n_ks? ( media-fonts/lohit-devanagari )
+	l10n_mai? ( media-fonts/lohit-devanagari )
+	l10n_ml? ( media-fonts/lohit-malayalam )
+	l10n_mr? (
+		media-fonts/lohit-devanagari
+		media-fonts/lohit-marathi
+	)
+	l10n_ne? (
+		media-fonts/lohit-devanagari
+		media-fonts/lohit-nepali
+	)
+	l10n_or? ( media-fonts/lohit-odia )
+	l10n_pa? ( media-fonts/lohit-gurmukhi )
+	l10n_ru? (
+		media-fonts/font-misc-cyrillic
+		media-fonts/oldstandard
+		media-fonts/paratype
+		media-fonts/paratype-astra
+	)
+	l10n_sa? ( media-fonts/lohit-devanagari )
+	l10n_sat? ( media-fonts/lohit-devanagari )
+	l10n_sd? ( media-fonts/lohit-devanagari )
+	l10n_si? ( media-fonts/lklug )
+	l10n_syc? ( media-fonts/font-misc-meltho )
+	l10n_ta? (
+		media-fonts/lohit-tamil
+		media-fonts/lohit-tamil-classical
+	)
+	l10n_te? ( media-fonts/lohit-telugu )
+	l10n_th? ( media-fonts/thaifonts-scalable )
+	latin? (
+		media-fonts/cantarell
+		media-fonts/courier-prime
+		media-fonts/dejavu
+		media-fonts/droid
+		media-fonts/font-bh-ttf
+		media-fonts/font-cursor-misc
+		media-fonts/liberation-fonts
+		media-fonts/noto
```

Comment:
How did you choose which fonts to use? I see you replaced hack and source-pro with other fonts

Reply:
My wish for this ebuild is to have a good selection of reasonably popular fonts, and to render all the glyphs correctly, (provided that the correct language USE-flag is set).

I don't know how to measure popularity of fonts. I personally use hack in terminal, and can readd it back to my world file when this PR is merged. Tbh, I thought hack is pretty popular (in my social bubble), so I probably wouldn't remove it yet. But that goes to my preference, ignoring preference of other users.

Maybe install all the popular fonts, and let users choose which font to use among the installed ones? After all, users always have a choice to install the wanted fonts manually, and ignore this meta package if they want to minimize the list of fonts installed

Reply:
> My wish for this ebuild is to have a good selection of reasonably popular fonts, and to render all the glyphs correctly, (provided that the correct language USE-flag is set).

Yep, agree there.

> 
> I don't know how to measure popularity of fonts. I personally use hack in terminal, and can readd it back to my world file when this PR is merged. Tbh, I thought hack is pretty popular (in my social bubble), so I probably wouldn't remove it yet. But that goes to my preference, ignoring preference of other users.
> 

I just googled "popular (monospace) fonts" and hack was rarely represented in those lists. 

> Maybe install all the popular fonts, and let users choose which font to use among the installed ones? 

If I understand you correctly here, that crossed my mind, but I'd rather leave the choice for users to begin with. This way you're not installing 100 fonts where you use 1. This is especially true with monospace fonts. 
What do you think about this dirty solution:
```
|| (
	monospace-font-1
	monospace-font-2
	monospace-font-3
	monospace-font-4
)
```

We'd add this to the very core, or under "latin".

> After all, users always have a choice to install the wanted fonts manually, and ignore this meta package if they want to minimize the list of fonts installed

Indeed, but I believe we should attempt to provide reasonable defaults here. I thought about including pretty much ~every font under media-fonts/* in this ebuild one way or another (`extra` use flag for example), but it quickly became a stupid idea, since users can choose to install whatever extra they want.

Reply:
>  What do you think about this dirty solution:

That works. In that case the first font in the list should have distinguishable 0Oo, lI1|. Not all fonts you selected for the "new" list satisfy that.

> We'd add this to the very core, or under "latin".

I'd say core. Having *a* monospace font (which is not necessary latin-only) is useful even without fonts specific to latin.

> ~every font under media-fonts/*

I actually had tried this for a while, the results were bad. Some wild font (don't know which one) was causing rendering issues by declaring that it has all the glyphs, but actually having only some. And there was a case where some programs couldn't decide which font to use to render certain glyphs, and randomly were switching between fonts